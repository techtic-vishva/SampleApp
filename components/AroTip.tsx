import React from 'react'
import { StyleSheet, Image } from 'react-native'
import { Text } from './Themed'

export default function (props: { message: string }) {
	return (
		<Text style={styles.container}>
			{/* Necessary space for center alignment to work */}
			&nbsp;
			<Image source={require('../assets/images/arobot-1.png')} style={styles.image} />
			{/* Padding between the image and text */}
			&nbsp;&nbsp;&nbsp;&nbsp;
			<Text style={styles.title}>{props.message}</Text>
		</Text>
	)
}

const styles = StyleSheet.create({
	container: {
		width: '80%',
		textAlign: 'center',
		marginVertical: 30
	},
	image: {
		width: 20,
		height: 20,
		resizeMode: 'contain'
	},
	title: {
		fontFamily: 'objektiv-md-italic',
		fontSize: 15
	}
})
