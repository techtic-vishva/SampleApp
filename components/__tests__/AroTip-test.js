import * as React from 'react'
import renderer from 'react-test-renderer'
import AroTip from '../AroTip'

it('render when passing the props', () => {
	const tree = renderer.create(<AroTip message="" />).toJSON()
	expect(tree).toMatchSnapshot()
})
