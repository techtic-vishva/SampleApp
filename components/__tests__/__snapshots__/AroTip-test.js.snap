// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render when passing the props 1`] = `
<Text
  style={
    Array [
      Object {
        "color": "#fff",
        "fontFamily": "objektiv",
      },
      Object {
        "marginVertical": 30,
        "textAlign": "center",
        "width": "80%",
      },
    ]
  }
>
   
  <Image
    source={1}
    style={
      Object {
        "height": 20,
        "resizeMode": "contain",
        "width": 20,
      }
    }
  />
      
  <Text
    style={
      Array [
        Object {
          "color": "#fff",
          "fontFamily": "objektiv",
        },
        Object {
          "fontFamily": "objektiv-md-italic",
          "fontSize": 15,
        },
      ]
    }
  >
    
  </Text>
</Text>
`;
